pkg/kubelet/pluginmanager/cache/actual_state_of_world.go

```diff
@@ -47,7 +48,7 @@ type ActualStateOfWorld interface {
 	// because the plugin should have been unregistered in the reconciler and therefore
 	// removed from the actual state of world cache first before adding it back into
 	// the actual state of world cache again with the new timestamp
-	AddPlugin(pluginInfo PluginInfo) error
+	AddPlugin(ctx context.Context, pluginInfo PluginInfo) error
```

Comment:
the usual concern is about adding a parameter just and only to the affected function and not to the rest of the interface. Did we agree about a way forward? I can't recall.
Note this is not-blocking comment as I don't think there is a clearly better way forward.